packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -412,3 +407,36 @@ class _RefreshOnGCToggleButton extends StatelessWidget {
     );
   }
 }
+
+class _ProfileHelpLink extends StatelessWidget {
+  const _ProfileHelpLink({Key? key}) : super(key: key);
+
+  static const _documentationTopic = 'profile';
```

Comment:
let's make this 'allocationProfile' to make it more specific. that will be easier to understand when looking at our analytics data since 'profile' by itself is pretty general